File: Modul-2/PYTHON/PRAK204_2410817210001_MuhammadFadillah.py
r = float(input("Masukan Jari-jari : "))
t = float(input("Masukan Tinggi : "))

volume = 3.1428571428571 * r * r * t
luas = 2 * 3.1428571428571 * r * (r + t)
keliling = 2 * 3.1428571428571 * r

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: Modul-2/C/PRAK204_2410817210001_MuhammadFadillah.c
#include <stdio.h>

int main() {
    float r, t, volume, luas, keliling;

    printf("Masukan jari-jari : ");
    scanf("%f", &r);
    printf("Masukan tinggi : ");
    scanf("%f", &t);

    volume = 3.1428571428571 * r * r * t;
    luas = 2 * 3.1428571428571 * r * (r + t);
    keliling = 2 * 3.1428571428571 * r;

    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);

    return 0;
}